packages/ffe-datepicker-react/src/calendar/Calendar.js

```diff
@@ -163,16 +162,27 @@ export default class Calendar extends Component {
     }
 
     renderDate(date, index) {
-        if (date.isLead) {
-            return <LeadDate key={date.date} date={date} />;
+        const { calendar } = this.state;
+
+        if (date.isNonClickableDate) {
+            return <NonclickableDate key={date.date} date={date} />;
         }
+
+        const forceFocus =
+            document.activeElement !== this.nextMonthButtonElementRef.current &&
+            document.activeElement !== this.prevMonthButtonElementRef.current;
+
         return (
-            <ActiveDate
+            <ClickableDate
                 date={date}
+                month={calendar.focusedMonth()}
                 headers={`header__${this.datepickerId}__${index}`}
                 key={date.date}
                 onClick={this.mouseClick}
-                forceFocus={this.forceDateFocus}
+                forceFocus={forceFocus}
```

Comment:
Hadde det holdt å bruke `isFocusingHeader` her? Altså hvis vi fjerner `forceFocus` helt (fjern linje 171-173 og linje 182), kaller `isBlured` for `isFocusingHeader` og så kan `ClickableDate` sin `focusIfNeeded` være
```


    focusIfNeeded() {
        const { date, isFocusingHeader, ..... } = this.props;
        if (date.isFocus && !isFocusingHeader) {
            .....
        }
    }
```